knife/lib/chef/knife/bootstrap/templates/chef-license.erb

```diff
@@ -0,0 +1,776 @@
+#!/bin/sh
```

Comment:
(If there are necessary breaking changes or high risk changes, we should bump version major or similar semantic communication)

Reply:
Hi @tpowell-progress,

When we started working on these changes, the `install.sh` endpoint was unavailable on the new download APIs. The architects suggested having a local copy of that file on the knife repo as a temporary solution.

Recently, the licensing team has added those endpoints and we thought of implementing it with the workstation licensing changes, But I've updated the code to use the new APIS.

To your second point, this is not a breaking or high-risk change. This will show a warning to the user to remind the necessity of getting a license and activating it, or otherwise, with Infra 19, things will not work as it is. 

Please review and let me know your thoughts.

Reply:
@ashiqueps Ok, so I see that `mixlib/install` is not included in chef. Looks like it might be a little more work than the quick and dirty solution here @jaymzh

Reply:
@ashiqueps looking at `mixlic-install` https://github.com/chef/mixlib-install/blob/main/lib/mixlib/install/generator/bourne.rb#L24 it looks like you can invoke one of the class methods or subclass/add a class method to generate what you want. This would save on duplication and prevent drift between the different scenarios.